osu.Game.Tests/Visual/SongSelectV2/TestSceneScreenFooter.cs

```diff
@@ -49,11 +48,11 @@ public void SetUp() => Schedule(() =>
                 },
             };
 
-            screenFooter.SetButtons(new ScreenFooterButton[]
+            screenFooter.SetButtons(new[]
             {
-                new FooterButtonMods(modOverlay) { Current = SelectedMods },
-                new FooterButtonRandom(),
-                new FooterButtonOptions(),
```

Comment:
I read the commit description and still don't understand *what* this is solving. "let's replace them" okay but why?

Reply:
I can't instantiate `FooterButtonOptions` anymore, it needs `SongSelect` instance. I can make that nullable but it means making a lot of other things nullable and becomes a hassle.

Reply:
can't you just add a song select to the test?